app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -11,12 +11,21 @@ def raw_api_endpoint(hostname, port)
     def kubernetes_connect(hostname, port, options)
       require 'kubeclient'
       api_endpoint = raw_api_endpoint(hostname, port)
-      kubeclient = Kubeclient::Client.new(api_endpoint, kubernetes_version)
       # TODO: support real authentication using certificates
-      kubeclient.ssl_options(:verify_ssl => OpenSSL::SSL::VERIFY_NONE)
-      kubeclient.basic_auth(options[:username], options[:password]) if options[:username] && options[:password]
-      kubeclient.bearer_token(options[:bearer]) if options[:bearer]
-      kubeclient
+      ssl_options = {:verify_ssl => OpenSSL::SSL::VERIFY_NONE}
+      auth_options = {}
+      if options[:username] && options[:password]
+        auth_options[:username] = options[:username]
+        auth_options[:password] = options[:password]
+      end
+      auth_options[:bearer_token] = options[:bearer_token] if options[:bearer]
```

Comment:
@moolitayer this should be:

    auth_options[:bearer_token] = options[:bearer] if options[:bearer]